src/fragments/lib/push-notifications/react-native/interact_with_notifications/notification-lifecycle.mdx

```diff
@@ -0,0 +1,29 @@
+## Notification lifecycle
+
+Before delving into details about the various functions Amplify provides, it can be helpful to better understand lifecycle of a notification as it moves through your app in its various states once you've integrated Amplify Push Notification.
+
+For the purposes of this guide, we will simplify the terminology as follows
+
+- **Native layer** - This is the native (i.e. iOS or Android) application layer on top of which React Native apps are built.
+- **Application layer** - This is your React Native app.
+
+### In a foreground state
+
+![Foreground state lifecycle](/images/push-notifications/cross-platform-lifecycle/pn-lifecycle-foreground.png)
+
+- Notifications are not displayed when received in foreground
+- It is possible for notifications to arrive in your user's notification center but not be interacted with until a later time, including while the app is in the active foreground
```

Comment:
this statement is not clear to me

Reply:
I updated the wording to be bit more verbose but hopefully more clear
```
- It is possible for notifications to arrive in your user's notification center but not be opened until a later time
- Since it is possible to interact with the notification center at any time on a device, your user could even open a notification while your app is in the active foreground
```

Reply:
maybe remove "active" @cshfang ?